scrapyjs/middleware.py

```diff
@@ -109,6 +115,9 @@ def process_request(self, request, spider):
             headers=new_headers,
         )
 
+        if self.splash_auth:
+            req_rep.headers['Authorization'] = self.splash_auth
```

Comment:
I think it is better to allow override Authorization header per-request even if splash_auth is set (see changes at https://github.com/scrapy-plugins/scrapy-splash/pull/41)